src/vault.rs

```diff
@@ -135,6 +144,20 @@ impl Client {
         Ok(())
     }
 
+    pub fn kubernetes_auth(&mut self, role: String) -> Result<(), Error> {
+        let mut file = File::open("/var/run/secrets/kubernetes.io/serviceaccount/token").expect("Unable to open");
+        let mut jwt = String::new();
+        file.read_to_string(&mut jwt);
+        // .is_err() {
+        //     return Err(Error::);
+        //     // return Err(String::from("Kubernetes authentication failed"));
+        // };
```

Comment:
Is this error handling not needed?

Reply:
I wasn't sure how to add it properly.. 
`read_to_string?` is not possible (perhaps, after Rust language changes)
and I could not find a usage `Error` class.. I guess, new one needs to be created